easybuild/easyconfigs/m/MultiQC/MultiQC-1.7-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,107 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+# Elements derived from work by Pablo Escobar
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
```

Comment:
@smoors Maybe add a comment about the need to set `locale` so at least it's documented somewhere?

Reply:
Not sure if this is the suitable place to document this though, since it's a problem related to `Click`, not to `MultiQC`...